tests/storm/bolt_test.py

```diff
@@ -20,6 +20,25 @@ def test_ack(self):
 
         mock_send_command.assert_called_once_with('ack', {'id': tup.id})
 
+    def test_heartbeat(self):
+        heartbeat = StormTuple(None, None, '__heartbeat', -1, [])
+
+        patches = mock.patch.multiple(self.instance, process_tuple=mock.DEFAULT,
```

Comment:
Looks good, but, since this code is not in a "real fixture" as for SimpleBolt, what about something like:

```
with mock.patch.multiple(
        self.instance,
        process_tuple=mock.DEFAULT,
        ack=mock.DEFAULT,
        send_command=mock.DEFAULT) as values:

    self.instance._process_tuple(heartbeat)
   
    values['send_command'].assert_called_once_with('sync')
    assert not values['process_tuple'].called
    assert not values['ack'].called
```